docs/ssh-keys/deploy-ssh-key.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Deploy SSH Key
+intro: Deploy your SSH Keys to each of the servers you want the keys on.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/ssh-keys/link-ssh-key
+    next:
+        - /docs/servers/ssh-into-server
+    guides:
+    related:
+    featured:
+required_permissions:
+    - application:deploy
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `SSH Keys` card
+1. Click `View` in the `SSH Keys` card to see a list of existing `SSH Keys`
+1. Click `DEPLOY` on the SSH Key you want to deploy
+1. Choose the pipeline to be deployed, then click `Next`
```

Comment:
Both sentences look semantically equal to me. In "Choose the pipeline to be deployed, then click `Next`", "the" does not mean a singular, specific pipeline.

Reply:
@brlipi yes, because although the first one is semantically correct, I believe the second one uses certain words to help guide the user to success.

> "Choose the pipeline to be deployed"

In this sentence, I understand that any pipeline I choose to deploy[1] will lead me to the article expected output: deploy an ssh key to the server.

> "Choose a deploy pipeline to be run"

In this case there's a highlight that it needs to be a "deploy pipeline".

[1] I know the current `DEPLOY` button may lead to some confusion, but we _deploy_ resources by _running_ pipelines, _deploy_ is a pipeline operation; in the docs, we try helping our users achieve success by using the same naming patterns as the current app version, making it easier to recognize.

Reply:
Understood.